flink-table/flink-sql-client/src/test/java/org/apache/flink/table/client/cli/CliTableauResultViewTest.java

```diff
@@ -167,8 +177,9 @@ public void testBatchResult() {
 				"|   false | -2147483648 |  9223372036854775807 |                     (NULL) |    12345.06789 |    2020-03-01 18:39:14.123 |\n" +
 				"|    true |         100 | -9223372036854775808 |                 abcdefg111 |         (NULL) | 2020-03-01 18:39:14.123456 |\n" +
 				"|  (NULL) |          -1 |                   -1 | abcdefghijklmnopqrstuvwxyz |   -12345.06789 |                     (NULL) |\n" +
+				"|  (NULL) |          -1 |                   -1 |         这是一段中文 |   -12345.06789 |      2020-03-04 18:39:14.0 |\n"	+
```

Comment:
Looks like we can not use UTF-8 length too...

Reply:
Hi, Jingsongli, 
Here's right,  the String  `'         这是一段中文 '`  has same length with `'abcdefghijklmnopqrstuvwxyz'` and shows well  when print in terminal. 
You can see the final view if you using  sql-client .

Reply:
You sure about this, because it doesn't look right when I print out the string via System.out.println in IDE

Reply:
It also doesn't seem right when I executed this test in my terminal by command line.

Reply:
I think Jingsong is right, we can't use `String.getBytes().length` here. For example, the result of `"这是一段中文".getBytes().length()` is 18, but if we want the display correct, it should be 12.

Reply:
The problem is that UTF-8 char's display width is different with UTF-8 char's bytes length. `"这是一段中文".getBytes().length()` will return 18 because length of Chinese char is 3 in UTF-8 unicode，here we can use GBK unicode which Chinese char's length. 
But GBK is not common enough than UTF-8.

Reply:
Using GBK works well with Chinese, 
![image](https://user-images.githubusercontent.com/5163645/76138195-74360880-6080-11ea-96ca-e1bc90c1dfdb.png)


my concern is other language like Japanese/Korean have same probelm.

Reply:
It seems this is a full width / half width character issue, but not necessarily an encoding problem.